### PR TITLE
Update links from github wiki to docserver

### PR DESCRIPTION
**Pull request by karenswry**
As described in T1419 - replaced links to Wiki with links to Docserver where it was necessary + brushed up a few things in the repository's Readme.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,40 +43,40 @@ Themis was designed to provide complicated cryptosystems in an easy-to-use infra
...
 | 🖥 WebAssembly | [WASM Howto](https://docs.cossacklabs.com/pages/js-wasm-howto/)| [docs/examples/js](https://github.com/cossacklabs/themis/tree/master/docs/examples/js) | [![npm](https://img.shields.io/npm/v/wasm-themis.svg)](https://www.npmjs.com/package/wasm-themis) |
-| 🐹 Go | [Go Howto](https://github.com/cossacklabs/themis/wiki/Go-HowTo)| [docs/examples/go](https://github.com/cossacklabs/themis/tree/master/docs/examples/go) ||
-| 🦀 Rust | [Rust Howto](https://github.com/cossacklabs/themis/wiki/Rust-Howto)| [docs/examples/rust](https://github.com/cossacklabs/themis/wiki/Rust-Howto) | [![crates](https://img.shields.io/crates/v/themis.svg)](https://crates.io/crates/themis)|
+| 🐹 Go | [Go Howto](https://docs.cossacklabs.com/pages/go-howto/)| [docs/examples/go](https://github.com/cossacklabs/themis/tree/master/docs/examples/go) ||
+| 🦀 Rust | [Rust Howto](https://docs.cossacklabs.com/pages/rust-howto/)| [docs/examples/rust](https://github.com/cossacklabs/themis/tree/master/docs/examples/rust) | [![crates](https://img.shields.io/crates/v/themis.svg)](https://crates.io/crates/themis)|
```
Comment (ilammy) [inline README.md]: > ```diff
> -[docs/examples/rust](https://github.com/cossacklabs/themis/wiki/Rust-Howto)
> +[docs/examples/rust](https://github.com/cossacklabs/themis/tree/master/docs/examples/rust)
> ```
Nice catch! 👍

File: tests/test.mk [hunk 2]
```diff
@@ -93,7 +93,7 @@ ifeq ($(NODE_VERSION),)
 	@echo 2>&1 "------------------------------------------------------------"
 	@echo 2>&1 "Node.js is not installed. Cannot run tests in Emscripten environment."
 	@echo 2>&1 ""
-	@echo 2>&1 "Please make sure you have \"node\" binary available in PATH and try again."
+	@echo 2>&1 "Make sure you have \"node\" binary available in PATH and try again."
```
Comment (ilammy) [inline tests/test.mk]: Rude! 😝

Comment (karenswry) [reply tests/test.mk]: Rude, but all the current style guides agree on "please" - that we should only use "please" when it's our fault and we're making the user go the extra mile to make things work due to an inconvenience/bug.

Comment (ilammy) [reply tests/test.mk]: Thanks, now I understand why.

**Pull request merged**